README.adoc

```diff
@@ -0,0 +1,32 @@
+= Spring Cloud Task
+
+Is a project centered around the idea of processing on demand.  A user is able to develop
+a “task” that can be deployed, executed and removed on demand, yet the result of the
+process persists beyond the life of the task for future reporting.
+
+
+== Requirements:
+
+* Java 7 or Above
+
+== Build:
+
+[source,shell,indent=2]
+----
+$ mvn clean install
```

Comment:
Why clean? Why install ?
What about `mvn package` ? I know we may be promoting clean install elsewhere, but there is no reason, really